pages/about/groups/agwg.md

```diff
@@ -33,67 +34,79 @@ Some information on this page is also shown on, and may be more current in, the
 {% include_cached toc.html type="end" %}
 {:/}
 
-## Announcements
+## Charter
 
-No announcements at the moment.
+The group is chartered until October 2025.
 
-## Current Work
+To learn about the group’s current focus, scope and deliverables, see the [AG Working Group Charter](https://www.w3.org/WAI/GL/charter).
 
-The Working Group maintains a comprehensive list of [publications and current timelines](https://www.w3.org/WAI/GL/wiki/Timelines).
+## Current work
 
-The [Project Plan](https://www.w3.org/WAI/GL/wiki/Timelines) details intended timeline and milestones for this work. A [list of publications](https://www.w3.org/TR/tr-groups-all#tr_Web_Content_Accessibility_Guidelines_Working_Group) on the W3C Technical Reports page includes completed deliverables that are no longer worked on by the Working Group.
+The work of the AG Working Group follows the [W3C Process](https://www.w3.org/2023/Process-20231103/). 
```

Comment:
I am not sure when this crept in. It is in the current Admin section although I am not sure what purpose it serves. Hmmm 🤔... 🤔... 🤔...  I don't think it is necessary so just remove it. That would go for any other instances of it too.